recipe/meta.yaml

```diff
@@ -19,17 +19,18 @@ requirements:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - crossenv                            # [build_platform != target_platform]
-    - maturin >=0.14,<0.15                # [build_platform != target_platform]
+    - maturin >=1.2.1,<2                  # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
+    - rust 1.73.0.dev20230713
```

Comment:
this line probably needs to be reconciled with the line above it no?

Reply:
i switched to using rust_compiler_version